src/rest/domain/Stream.js

```diff
@@ -46,17 +46,27 @@ export default class Stream {
     }
 
     async hasPermission(operation, userId) {
+        // eth addresses may be in checksumcase, but userId from server has no case
+        const userIdCaseInsensitive = typeof userId === 'string' ? userId.toLowerCase() : userId
```

Comment:
what else could it be besides string and null? Maybe at least add comment in what cases you expect it go through as "just" userId

Reply:
forced to undefined if failing typeof string in 613ea4a